keras_nlp/src/tests/test_case.py

```diff
@@ -455,6 +455,49 @@ def run_backbone_test(
         if run_quantization_check and has_quantization_support():
             self.run_quantization_test(backbone, cls, init_kwargs, input_data)
 
+    def run_cv_backbone_test(
```

Comment:
How is this different than the other backbone test? Probably we should consolidate.

Reply:
I wanted to skip some of the tokenizer and  input sequence length related tests. I will consolidate it into one with a cv flag